src/popup/components/AppSettings.vue

```diff
@@ -76,6 +83,7 @@ export default {
       showSLButton: false,
       positionSLButton: "right-inside",
       reportURISLButton: "",
+      extension_version: "",
```

Comment:
Nits:
```suggestion
      extension_version: "development",
```
Just as a precaution to avoid showing empty string in the header